endpoint/models/endpoint_mixin.py

```
@@ -238,14 +241,18 @@ def _get_handler(self):
                 _("Missing handler for exec mode %s") % self.exec_mode
             ) from e
 
-    def _handle_request(self, request):
+    def _handle_request(self, request, querystring_params=None):
         # Switch user for the whole process
         self_with_user = self
         if self.exec_as_user_id:
             self_with_user = self.with_user(user=self.exec_as_user_id)
         handler = self_with_user._get_handler()
         try:
-            res = handler(request)
+            # In case the handler does not support params
```

I would simply let it fail. Or if you really want you can catch a `TypeError` and log it. But I would just let it break. Beside, I doubt anyone did ever implement a different handler :)